.gitignore

```diff
@@ -2,3 +2,4 @@ target/**
 */target/**
 **/schema/**
 tarpaulin-report.html
+*.profraw
```

Comment:
Is this generated by something in project, or your private file? If your private thingy, then it shall go to your local ignored list

Reply:
this is generated when running tarpaulin with llvm engine